tests/functional/grpo_rm_env.sh

```
@@ -0,0 +1,47 @@
+    policy.train_micro_batch_size=1 \
+    cluster.gpus_per_node=2 \
+    cluster.num_nodes=1 \
+    env.reward_model.model_name=Skywork/Skywork-Reward-V2-Qwen3-0.6B \
+    env.reward_model.resources.gpus_per_node=1 \
+    env.reward_model.resources.num_nodes=1 \
+    logger.tensorboard_enabled=true \
+    logger.log_dir=$LOG_DIR \
+    logger.wandb_enabled=false \
+    logger.monitor_gpus=true \
+    checkpointing.enabled=false \
+    $@ \
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)